examples/config/_common/log4j2.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+  ~ Druid - a distributed column store.
+  ~ Copyright 2012 - 2015 Metamarkets Group Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<Configuration status="WARN">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{ISO8601} %p [%t] %c - %m%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <Logger name="io.druid.curator.inventory.CuratorInventoryManager" level="warn"/>
+        <Logger name="io.druid.client.BatchServerInventoryView" level="warn"/>
+        <Logger name="io.druid.client.ServerInventoryView" level="warn"/>
+        <Logger name ="com.metamx.http.client.pool.ChannelResourceFactory" level="warn"/>
```

Comment:
IMHO this should simply be a copy of https://github.com/druid-io/druid/blob/master/common/src/main/resources/log4j2.xml